.circleci/config.yml

```diff
@@ -71,13 +88,13 @@ jobs:
       - checkout
       - run: 
           name: Build docker image
-          command: docker build -t klaatu:latest -f Dockerfile.desktop .
+          command: docker build -t klaatu:firefox-release -f Dockerfile.desktop .
```

Comment:
I assume the default `FIREFOX_VERSION` is `-release` or something? (since I noticed we were passing `--build-arg="FIREFOX_VERSION=-(nightly|beta)` above on L57 and L74 respectively.

Reply:
Correct